docs/sources/get-started/_index.md

```diff
@@ -0,0 +1,94 @@
+| Refer to our style guide                       | [Style guide]({{< relref "../write/style-guide/style-conventions" >}})                                                                                                                           |
+
+{{% /responsive-table %}}
+
+### 4. Review the docs
+
+{{% responsive-table %}}
+
+| Action                                 | Considerations                                                                                                                                                                                      |
+| -------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Build the docs locally                 | See the [build locally guidelines]({{< relref "../review/run-a-local-webserver" >}}) and check for errors, particularly in linking.                                                                 |
+| Open a PR and tag the docs team        | Use the label type/docs.                                                                                                                                                                            |
+| Use the docs validator and vale linter | These are automatic checks from within the PR itself. Learn more about the [docs-validator]({{< relref "../review/doc-validator" >}}) and the [Vale linter]({{< relref "../review/lint-prose" >}}). |
+
+{{% /responsive-table %}}
+
+### 5. Publish the docs
+
+{{% responsive-table %}}
+
+| Action                                       | Considerations                                                                                                                                                                    |
+| -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Review publishing options                    | Engage #docs to discuss the publishing options for different stages in the release cycle.                                                                                         |
+| Backport, if required                        | See the [backporting guidelines]({{< relref "../review/backporting" >}}).                                                                                                         |
+| Add What’s New or release notes, if required | See the [Contribute to release notes guidelines]({{< relref "../contribute-documentation/contribute-release-notes" >}}). \*In Grafana, use the label add-to-whats-new in your PR. |
+
+{{% /responsive-table %}}
```

Comment:
@jdbaldry This one change from prettier is preventing the PR from passing. I can't commit the change because there's nothing in the code that is changed. Running prettier PR locally on this branch returns no changes. Any suggestions?

Reply:
I bet that it is a missing trailing newline at the end of the file.

Reply:
I think the suggestion is improperly provided. I'll fix it up to unblock the PR.